TypeApprovalRegister+services.yaml

```diff
@@ -155,10 +198,12 @@ clients:
       application-name: ExecutionAndTraceLog
       release-number: 1.0.0
       uuid: tar-1-0-0-http-c-0040
+            tar-1-0-0-http-c-bm-eatl-1-0-0-000
```

Comment:
As per Structure of uuid definition. api-segment for eatl http-c and tcp-c would be "bs", since we have only one op-c under EatL that too "bs"